src/tests/ftest/security/cont_acl.py

```diff
@@ -91,26 +91,25 @@ def test_container_user_acl(self):
                       "   base_acl_entries= %s\n", base_acl_entries)
         self.add_pool()
         secTestBase.create_acl_file(acl_file_name, base_acl_entries)
-        self.container_uuid = self.create_container_with_daos(
-            self.pool, None, acl_file_name)
+        self.container = self.create_container_with_daos(self.pool, None, acl_file_name)
 
         # (3)Verify container permissions rw, rw-attribute
         permission_type = "attribute"
         self.log.info("(3)==>Verify container permission %s", permission_type)
 
-        self.update_container_acl(
-            secTestBase.acl_entry(test_user_type, test_user, "rw"))
+        with self.container.daos.temp_exit_status_exception(False):
```

Comment:
@phender Do you think it would be better to have a wrapper `TestContainer.temp_exit_status_exception`, so this:
```
with self.container.daos.temp_exit_status_exception(False):
```
becomes this?
```
with self.container.temp_exit_status_exception(False):
```

The wrapper would just be:
```
class TestContainer():
    def temp_exit_status_exception(self, tmp_value):
        return self.daos.temp_exit_status_exception(tmp_value)
```

Reply:
I think a having a `TestContainer.no_exception()` method to call the `ExecutableCommand.no_exception()` context manager would be easier to use/understand, e.g.
```
        with self.container.no_exception():
```